en/news/_posts/2018-02-24-ruby-2-6-0-preview1-released.md

```diff
@@ -18,50 +18,51 @@ Ruby 2.6 introduces an initial implementation of JIT (Just-in-time) compiler.
 
 JIT compiler aims to improve performance of any Ruby program execution.
 Unlike ordinary JIT compilers for other languages, Ruby's JIT compiler does JIT compilation in a unique way, which prints C code to a disk and spawns common C compiler process to generate native code.
-See also: https://github.com/vnmakarov/ruby/tree/rtl_mjit_branch#mjit-organization
+See also: [Vladimir Makarov's mjit-organization branch](https://github.com/vnmakarov/ruby/tree/rtl_mjit_branch#mjit-organization).
 
-How to use: Just specify "--jit" in command line or $RUBYOPT environment variable.
-Specifying "--jit-verbose=1" allows to print basic information of ongoing JIT compilation. See "ruby --help" for other options.
+How to use: Just specify `--jit` in command line or `$RUBYOPT` environment variable. Specifying `--jit-verbose=1` allows to print basic information of ongoing JIT compilation. See `ruby --help` for other options.
 
 The main purpose of this JIT release is to provide a chance to check if it works for your platform and to find out security risks before the 2.6 release.
 Currently JIT compiler is supported only when Ruby is built by gcc or clang and the compiler is available on runtime. Otherwise you can't use it for now.
 
-As of 2.6.0-preview1, we're just preparing infrastructure for JIT and very few optimizations are implemented.
-You can measure some of potential improvements in micro benchmarks with this release, but it is NOT ready for benchmarking final performance of Ruby's JIT compiler, especially for larger programs like Rails applications.
+As of 2.6.0-preview1, we're just preparing infrastructure for JIT and very few optimizations are implemented. You can measure some of potential improvements in micro benchmarks with this release, but it is NOT ready for benchmarking final performance of Ruby's JIT compiler, especially for large programs like Rails applications.
 
 We're going to implement method iniling in JIT compiler, which is expected to increase Ruby's performance in order of magnitude.
```

Comment:
iniling => inlining

which is expected to increase Ruby's performance **by an** order of magnitude.

Personally, I would just simplify to "significantly", it's too hard to predict. No inlining is an optimization barrier, but it doesn't tell how well the larger compilation will optimize better.
cc @k0kubun

Reply:
> Personally, I would just simplify to "significantly", it's too hard to predict.

I'm okay with it. Thank you for the correction.

> No inlining is an optimization barrier, but it doesn't tell how well the larger compilation will optimize better.

Sorry, I couldn't understand what you want to mean here. Could you rephrase?

As you know, the background of expectation is that inlining allows to widen the scope to be optimized by C compiler. Do you mean this should describe about it?

Reply:
Sorry, I wrote that sentence in a rather unclear way.
Inlining enables other optimizations: it makes it possible to compile a wider scope/more code and more optimizations might trigger on that larger code.
But in general it's hard to predict how much performance inlining gains, because the gains essentially depend on other optimizations (if they trigger) and how much the heuristic decides to inline of course.

By that sentence I was trying to justify `significantly` is safe but `an order of magnitude` feels like it could be wrong. At least, I expect inlining alone will not gain that much. I expect quite a few other tweaks in  the current MJIT will be needed to achieve a 10x speedup on general Ruby code.
As an extreme example,  inlining on MJIT microbenchmarks without method/block calls cannot gain anything.

> Do you mean this should describe about it?

No, I think this news should stay simple, so I was just suggesting changing `by an order of magnitude` to `significantly`.

Reply:
That makes sense. Totally agreed.
@JuanitoFatas Could you include his suggestion too?